docs/self-managed/tasklist-deployment/tasklist-configuration.md

```diff
@@ -62,15 +62,15 @@ For those running a Self-Managed Camunda 8 environment, configuring multi-tenanc
 To ensure seamless integration and functionality, the multi-tenancy feature should also be enabled across all associated components. This is done using their specific multi-tenancy feature flags.
 :::
 
-## Elasticsearch
+## Elasticsearch or OpenSearch
 
-Tasklist stores and reads data in/from Elasticsearch.
+Tasklist stores and reads data from Elasticsearch or OpenSearch.
 
 ### Settings to connect
 
 Tasklist supports [basic authentication](https://www.elastic.co/guide/en/elasticsearch/reference/7.12/setting-up-authentication.html) for Elasticsearch. Set the appropriate username/password combination in the configuration to use it.
 
-#### Settings to connect to a secured Elasticsearch instance
+#### Settings to connect to a secured Elasticsearch or OpenSearch instance
```

Comment:
We can also specify that by default `Tasklist` application is initially configured to work with  `ElasticSearch`.
To modify this behavior and switch to OpenSearch, users can set the `camunda.tasklist.database` property to `opensearch`.

Furthermore, it would be beneficial to include a table similar to the one we have for ElasticSearch, outlining the necessary settings to connect to OpenSearch. There we can describe properties such as `camunda.tasklist.opensearch.url=http://localhost:9205`, `camunda.tasklist.opensearch.clusterName=opensearch-cluster`, and other relevant parameters, along with their default values.

Reply:
Thanks @ce-dmelnych I added the configurations :)